provisionerd/provisionerd.go

```diff
@@ -135,11 +135,13 @@ func NewMetrics(reg prometheus.Registerer) Metrics {
 				Namespace: "coderd",
 				Subsystem: "provisionerd",
 				Name:      "jobs_current",
+				Help:      "The number of concurrently running provisioner jobs.",
```

Comment:
Suggestion: Feel free to ignore, but it feels more future proof (and matches metric name) to not say too much about _how_ the job is running. 😄

```suggestion
				Help:      "The number of currently running provisioner jobs.",
```

Reply:
This is wise advice, thanks 😄 !